lua/ulx/modules/sh/timed_gag.lua

```diff
@@ -124,6 +124,14 @@ local function removeExpiredGagFromDatabase( ply )
     return SQLAction( "Deleting expired time gag", query, ply )
 end
 
+local function removeGagFromDatabase( ply )
```

Comment:
removeExpiredGag could be replaced by this function unless logging `Deleting expired time gag` vs `Manually deleting time gag` is important
they run the same  query so maybe just a single function with the message `Deleting  timed gag` would be better